README.md

```diff
@@ -216,7 +216,7 @@ _This will use an in-memory implementation of the Fluid server to sync between t
 _This will run the local Fluid server implementation we call "Tinylicious", so you can sync between multiple browser
 instances._
 
-First, start Tinylicious by running these commands from `/server/tinylicious`:
+First, start Tinylicious by running these commands from `/server/routerlicious/packages/tinylicious`:
```

Comment:
Actually, this should be a slightly bigger change in this file... `npm install` in the tinylicious folder is not correct anymore (already wasn't, so this doc was outdated), we need to start with `pnpm install` in `server/routerlicious`, then the other two commands in the tinylicious folder. Since we're fixing other bits about t9s, let's fix that bit too.

Reply:
Documentation changed to run `pnpm install` in `server/routerlicious` and then the remaining commands in the `server/routerlicious/packages/tinylicious`.